.travis.yml

```diff
@@ -20,9 +20,9 @@ jobs:
   include:
     - stage: 'Lint'
       install:
-        - go get github.com/golangci/golangci-lint/cmd/golangci-lint
+        - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.19.0
```

Comment:
What are these changes for? They seem unrelated to this PR. If so, please put it in another PR.

Reply:
linter was causing problem, yeah this should be in a separate PR.

Reply:
Have created a separate PR for this linter-fix now.